homeassistant/components/sinch/notify.py

```diff
@@ -61,12 +61,10 @@ def __init__(self, config):
     def send_message(self, message="", **kwargs):
         """Send a message to a user."""
         targets = kwargs.get(ATTR_TARGET, self.default_recipients)
-        data = kwargs.get(ATTR_DATA, {})
+        data = kwargs.get(ATTR_DATA)
```

Comment:
This is enough. Please revert other changes.
```py
data = kwargs.get(ATTR_DATA) or {}
```

Reply:
Sorry for the late response, but this is now pushed.